keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1590,6 +1979,7 @@ For each of them, fill in the following information by copying the below templat
 - 2022-08-02: PR "Append new pod conditions when deleting pods to indicate the reason for pod deletion" merged
 - 2022-08-02: PR "Add worker to clean up stale DisruptionTarget condition" merged
 - 2022-08-04: PR "Support handling of pod failures with respect to the configured rules" merged
+- 2022-09-09: Bugfix PR for test "Fix the TestRoundTripTypes by adding default to the fuzzer" merged
```

Comment:
Also provide a summary of the sections of the KEP that changed for beta

Reply:
Done

Reply:
We aren't usually that verbose, but thanks :)